tests/fuzz_tests.rs

```diff
@@ -40,3 +40,12 @@ fn inf_loop() {
         true
     });
 }
+
+// https://github.com/image-rs/image-tiff/issues/33
+#[test]
+fn divide_by_zero() {
+    test_directory("./tests/fuzz_images/divide_by_zero", |file| {
+        decode_tiff(file).ok();
```

Comment:
This call to `Result::ok` doesn't seem to achieve anything.

Reply:
Well it removes the `unused result` error, but other than that it doesn't have any real value.

Reply:
The idiomatic way to express that is `let _ = decode_tiff(file);` in that case.

Reply:
You are right, fixed this one and the others in the same file.